R/validate_ensemble_inputs.R

```diff
@@ -168,9 +171,12 @@ validate_compound_taskid_set <- function(model_out_tbl,
     )
   }
 
-  dependent_tasks <- setdiff(task_id_cols, compound_taskid_set)
+  dependent_tasks <- setdiff(task_id_cols, c(compound_taskid_set, derived_tasks))
```

Comment:
I think the logic here and below is correct, I was just confused by the name because some of the things in `derived_tasks` may be task id variables for which a model's samples capture dependence. Adding a comment here just to orient the reader to what this is intended to include.
```suggestion
  # the set of "non-derived" task id variables for which the samples should capture dependence
  dependent_tasks <- setdiff(task_id_cols, c(compound_taskid_set, derived_tasks))
```

Reply:
I've taken the suggestion, but do you also think it would be good to rename this variable? Also any suggestions for a different name?